app/controllers/reminders_controller.rb

```diff
@@ -3,7 +3,7 @@ class RemindersController < ApplicationController
 
   def index
     authorize Reminder
-    @reminders = Reminder.all
+    @reminders = Reminder.assignment
```

Comment:
since we don't have the `Reminder.all` here, what will be shown at the `"normal" Reminders Index` ?

Reply:
The 'normal' index now will show reminders that concern assignments excluding the ones referring to the trial period. When the assignment reminders are produced through the cron job `kind` is set to `assignment`, when the trial period are produced `kind` is set to `trial`. So  `Reminder.assignment` and `Reminder.trial` work similarly to scopes.

Reply:
👍